src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -933,6 +933,26 @@ CHIP_ERROR ThermostatAttrAccess::AppendPendingPreset(Thermostat::Delegate * dele
         return CHIP_IM_GLOBAL_STATUS(ConstraintError);
     }
 
+    uint8_t totalCount = CountNumberOfPendingPresets(delegate);
```

Comment:
This conflicts with https://github.com/project-chip/connectedhomeip/pull/34936.  That needs to land, then we can merge this (much smaller) change on top.

Reply:
rewrote